src/host/_stream.cpp

```diff
@@ -317,9 +317,9 @@ constexpr unsigned int LOCAL_BUFFER_SIZE = 100;
 // - pcb - On input, number of bytes to write.  On output, number of bytes written.
 // - pcSpaces - On output, the number of spaces consumed by the written characters.
 // - dwFlags -
-//      WC_DESTRUCTIVE_BACKSPACE backspace overwrites characters.
-//      WC_KEEP_CURSOR_VISIBLE   change window origin desirable when hit rt. edge
-//      WC_ECHO                  if called by Read (echoing characters)
+//      WC_DESTRUCTIVE_BACKSPACE    backspace overwrites characters.
+//      WC_KEEP_CURSOR_VISIBLE      change window origin desirable when hit rt. edge
+//      WC_PRINTABLE_CONTROL_CHARS  if called by Read (echoing characters)

```

Comment:
fix the right half of the comment too, plx

Reply:
BAH.

Reply:
okay